checkov/sca_package_2/scanner.py

```diff
@@ -43,7 +44,8 @@ def run_scan(self) -> bool:
                 "path": bc_integration.repo_path,
                 "repoId": bc_integration.repo_id,
                 "id": bc_integration.timestamp,
-                "repositoryId": ""
+                "repositoryId": "",
+                "enableDotnetCpm": os.getenv('ENABLE_DOTNET_CPM', 'False'),
```

Comment:
all env vars in checkov are handled in `checkov/common/util/env_vars_config.py`, please add it there and use from import

Reply:
fixed, thank you!